code/player/TTTPlayer.Shop.cs

```diff
@@ -10,13 +15,85 @@ public enum BuyError
         None,
         InventoryBlocked,
         NotEnoughCredits,
-        RoleRestriction
+        NoAccess
+    }
+
+    public class Shop
+    {
+        public List<ShopItemData> Items { set; get; } = new();
```

Comment:
Can this just be initialized in the constructor?

Reply:
Why? We never initializing in the constructor :D